src/models/ipdCourse.js

```diff
@@ -0,0 +1,60 @@
+const { Model } = require('sequelize');
+
+export default (sequelize, DataTypes) => {
+  class IpdCourse extends Model {
+    static associate(models) {
+      // IpdCourse.belongsTo(models.Objective, { foreignKey: 'objectiveId', as: 'objective' });
+      // eslint-disable-next-line max-len
+      // IpdCourse.belongsTo(models.ActivityReportObjective, { foreignKey: 'activityReportObjectiveId', as: 'activityReportObjective' });
```

Comment:
dont put these here, it helps sequencing when all the associations introduced by a model are in that models file, not spread across all the other models